tools/OpenConsole.psm1

```diff
@@ -31,18 +31,17 @@ function Import-LocalModule
 
     if (-not (Test-Path "$modules_root\$Name")) {
         Write-Verbose "$Name not downloaded -- downloading now"
-        $module = Find-Module '$Name'
+        $module = Find-Module "$Name"

```

Comment:
Nit: again, the quotes here aren't necessary. Even with spaces (though not supported for module names, IIRC), the variable still represents a single string token. This isn't like passing a path with spaces to an application command that splits on spaces unless quotes are present. This doesn't hurt, but is completely unnecessary.

Reply:
```suggestion
        $module = Find-Module $Name
```